build/proxies-scenarios.yml

```diff
@@ -50,20 +49,23 @@ parameters:
   type: object
   default:
     - displayName: http
-      arguments: --variable protocol=http --property protocol=http
+      arguments: --variable serverScheme=http --variable downstreamScheme=http --property protocol=http
+    - displayName: https
+      arguments: --variable serverScheme=https --variable downstreamScheme=https --property protocol=https
```

Comment:
Should we add a new property for `downstreamProtocol` as I assume we'll also do some benchmarks with http here

Reply:
I was considering that, I didn't want to create too many variations though. This would quadruple the variations instead of double them. Are we okay with that?

Reply:
We don't have to run it now, and we'll maybe never do it, but saving the info is free. At least you supported it with the variables. We know that people will ask how much it cost to switch from http to https in the downstream.

Reply:
And also this way I can generate a legend that makes it obvious that it's https all the way. Instead of hard coding in the charts.